packages/vscode-ui5-language-assistant/src/extension.ts

```diff
@@ -63,6 +69,36 @@ export async function activate(context: ExtensionContext): Promise<void> {
     clientOptions
   );
 
+  // keep a reference to the current UI5 version
+  let currentVersion;
+
+  // create and register the command to open the SAPUI5 demokit
+  context.subscriptions.push(
```

Comment:
the activate method used to only handle the server process startup.
Now it also handles UI/UX concerns.

I would split these concerns to separate helper functions and avoid giving too much (direct) responsibility to the activate method.

Reply:
Ok, will be split in separate helpers to create the server and to create the statusbaritem